app/Http/Controllers/Front/NotFoundController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers\Front;
+
+use App\Http\Controllers\Controller;
+
+class NotFoundController extends Controller
+{
+    public function __invoke()
+    {
+        return view('errors.404');
```

Comment:
This is off. Looking at the code a view with back layout will be displayed at the front.

Lets create both a the `front.errors.404` and `back.errors.404` views so we can easily customise the front one in each project.

Reply:
The `errors.404` view actually did exactly that: 
`@include('errors.404-'.(request()->isBack() ? 'back' : 'front'))`

However I'll move the 404 views to the `front` and `back` directories and get rid of the `errors.404` view.

Reply:
👍